pootle/core/mixins/treeitem.py

```diff
@@ -679,8 +701,10 @@ def update_cache_job(instance):
     # Django should take care of setting this up, but it doesn't yet
     # (fixed in Django 1.10):
     # https://code.djangoproject.com/ticket/16734
-    script_name = (u'/' if settings.FORCE_SCRIPT_NAME is None
-                        else force_unicode(settings.FORCE_SCRIPT_NAME))
+    script_name = (
+        u'/'
+        if settings.FORCE_SCRIPT_NAME is None
+        else force_unicode(settings.FORCE_SCRIPT_NAME))
```

Comment:
IMHO the string should be on the first line and the rest should be vertically aligned with it.